From d1a9a51b996376fd764149d623c322b1ca31c948 Mon Sep 17 00:00:00 2001
From: Sergey Bolshakov <sbolshakov@altlinux.org>
Date: Sat, 15 Sep 2007 00:12:23 +0400
Subject: [PATCH 04/16] ALT nostdinc

---
 tcl/unix/configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcl/unix/configure.in b/tcl/unix/configure.in
index 0d3f426e2c..8893cf6d40 100755
--- a/tcl/unix/configure.in
+++ b/tcl/unix/configure.in
@@ -895,7 +895,7 @@ TCL_BUILD_STUB_LIB_PATH="`pwd`/${TCL_STUB_LIB_FILE}"
 TCL_STUB_LIB_PATH="${TCL_STUB_LIB_DIR}/${TCL_STUB_LIB_FILE}"
 
 # Install time header dir can be set via --includedir
-eval "TCL_INCLUDE_SPEC=\"-I${includedir}\""
+TCL_INCLUDE_SPEC=
 
 #------------------------------------------------------------------------
 # tclConfig.sh refers to this by a different name
-- 
2.21.0

