From 4a383af3e514f7a0d3282b15da838bf5e763cec5 Mon Sep 17 00:00:00 2001
From: "Vladimir D. Seleznev" <vseleznv@altlinux.org>
Date: Sun, 17 Sep 2017 07:10:07 +0300
Subject: [PATCH 09/17] ALT add "${libdir}/tcl" to package search path

---
 tcl/unix/configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcl/unix/configure.in b/tcl/unix/configure.in
index d37fe94a0d..dc2383c245 100755
--- a/tcl/unix/configure.in
+++ b/tcl/unix/configure.in
@@ -866,7 +866,7 @@ if test "$FRAMEWORK_BUILD" = "1" ; then
     test -z "$TCL_MODULE_PATH"  && \
 	TCL_MODULE_PATH="~/Library/Tcl /Library/Tcl /System/Library/Tcl"
 else
-    TCL_PACKAGE_PATH="${prefix}/share/tcl"
+    TCL_PACKAGE_PATH="${prefix}/share/tcl ${libdir}/tcl"
 fi
 
 #--------------------------------------------------------------------
-- 
2.21.0

