From d25c09dc09e8229b7465c21e5ddcbf7e247121a7 Mon Sep 17 00:00:00 2001
From: "Vladimir D. Seleznev" <vseleznv@altlinux.org>
Date: Sun, 17 Sep 2017 07:10:07 +0300
Subject: [PATCH 09/16] ALT add "${libdir}/tcl" to package search path

---
 tcl/unix/configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tcl/unix/configure.in b/tcl/unix/configure.in
index 080aecb3aa..08b92717eb 100755
--- a/tcl/unix/configure.in
+++ b/tcl/unix/configure.in
@@ -866,7 +866,7 @@ if test "$FRAMEWORK_BUILD" = "1" ; then
     test -z "$TCL_MODULE_PATH"  && \
 	TCL_MODULE_PATH="~/Library/Tcl /Library/Tcl"
 else
-    test -z "$TCL_PACKAGE_PATH" && TCL_PACKAGE_PATH="${prefix}/share/tcl"
+    test -z "$TCL_PACKAGE_PATH" && TCL_PACKAGE_PATH="${prefix}/share/tcl ${libdir}/tcl"
 fi
 
 #--------------------------------------------------------------------
-- 
2.21.0

